crates/forge/bin/cmd/verify/etherscan/mod.rs

```diff
@@ -512,7 +512,7 @@ mod tests {
                 &config,
             )
             .unwrap();
-        assert_eq!(client.etherscan_api_url().as_str(), "https://api-testnet.polygonscan.com/?/");
+        assert_eq!(client.etherscan_api_url().as_str(), "https://api-testnet.polygonscan.com/");
```

Comment:
Don't know what this is about

Reply:
this is outdated/no longer required, there were some cloudflare bugs with trailing /